src/Bones/babylon.skeleton.ts

```diff
@@ -232,7 +232,10 @@
                     }
                 }
 
-                bone.getInvertedAbsoluteTransform().multiplyToArray(bone.getWorldMatrix(), targetMatrix, index * 16);
+                if (bone._index !== -1) {

```

Comment:
No, that's not the intention. I'm trying to make it such that the skeleton can have more bones than the transformMatrix. There is an optimization we can make that will make the transformMatrix use less memory, but I don't have a good solution yet.